users/tests/views_test.py

```diff
@@ -0,0 +1,60 @@
+from django.urls import reverse
+from django.test import TestCase, Client
+from users.models import User
+
+
+user_data = {
+    'email': 'john@john.com',
+    'first_name': "John",
+    'last_name': "Doe",
+    'password': User.objects.make_random_password(),
+    'is_active': True
+}
+
+
+class LoginTestCase(TestCase):
+    """ Test case for user login view
+    """
+    def setUp(self):
+        self.client = Client()
+        User.objects.create_user(**user_data)
+        self.url = reverse('login')
+
+    def test_get_login_page(self):
+        """ Gets the login page
+        """
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+
+    def test_post_login_page(self):
+        """ Post data to login page
+        """
+        creds = {
+            'email':user_data['email'],
+            'password':user_data['password']
+        }
+
+        response = self.client.post(self.url, creds, follow=True)
+        self.assertEqual(response.status_code, 200)
```

Comment:
did you run the test? because i am sure that your code indicates that if the user has successfully logged in, it will be redirected so it's not http200